XCast/XCast.json

```diff
@@ -161,7 +161,7 @@
             }
         },
         "registerApplications": {
-            "summary": "Registers an application",
+            "summary": "Registers an application.\n  \n### Events \n\n No Events",
```

Comment:
This API allows to whitelist the apps which support dial service. If app want to dynamically update the app list same API should be called with updated list.

Reply:
Updated changes as per your review comments